_timeline/2017-12-10-multi-hop-wireless-test.md

```diff
@@ -0,0 +1,5 @@
+---
+title: First multi hop wireless test
+date: 2017-12-10
+---
+We tried to deploy a multi-hop wireless network using the prototype nodes to discover how well a wireless mesh works over a connection spread across a larger area. [Our Results](https://github.com/tomeshnet/documents/blob/master/technical/20171210_multi-hop-wireless-test.md)
```

Comment:
Would rephrase to something like "Deployed a multi-hop wireless network using our prototype nodes. We documented our findings on how well a wireless mesh works over a connection spread across a larger area (spoiler: ???)"